src/StructuredLogViewer.Avalonia/MainWindow.xaml.cs

```diff
@@ -566,7 +566,7 @@ private async Task BrowseForMSBuildExe()
             };
 
             var fileName = await openFileDialog.ShowAndGetFileAsync(this);
-            if (fileName != null)
+            if (ShouldNotExistFile(fileName))

```

Comment:
Let's not introduce a function and just call `!File.Exists` here, also remove the null check as it will check for null.

Reply:
Ah, I agree. Thank you